From c1d0ae8a614b4c99c78ec3e6634b093c16347d55 Mon Sep 17 00:00:00 2001
From: Ankit Agrawal <ankit.agarwal@intel.com>
Date: Thu, 4 Jan 2024 13:23:33 +0530
Subject: [PATCH] Removed Telephony module in IVI.

Since we are not supporting telephony module in AAOS, hence removing
it from image.

Tests: Device is booting fine and no crash observed.

Tracked-On: OAM-113415
Signed-off-by: Ankit Agrawal <ankit.agarwal@intel.com>
---
 target/product/base_system.mk        | 1 -
 target/product/default_art_config.mk | 1 -
 target/product/handheld_system.mk    | 1 -
 3 files changed, 3 deletions(-)

diff --git a/target/product/base_system.mk b/target/product/base_system.mk
index 14ce1af744..72a7301285 100644
--- a/target/product/base_system.mk
+++ b/target/product/base_system.mk
@@ -260,7 +260,6 @@ PRODUCT_PACKAGES += \
     task_profiles.json \
     tc \
     telecom \
-    telephony-common \
     tombstoned \
     traced \
     traced_probes \
diff --git a/target/product/default_art_config.mk b/target/product/default_art_config.mk
index 0fa9058f0b..044f78f646 100644
--- a/target/product/default_art_config.mk
+++ b/target/product/default_art_config.mk
@@ -41,7 +41,6 @@ PRODUCT_BOOT_JARS += \
     framework-minus-apex \
     framework-graphics \
     ext \
-    telephony-common \
     voip-common \
     ims-common
 
diff --git a/target/product/handheld_system.mk b/target/product/handheld_system.mk
index b7a2d0dad0..b52cadf593 100644
--- a/target/product/handheld_system.mk
+++ b/target/product/handheld_system.mk
@@ -67,7 +67,6 @@ PRODUCT_PACKAGES += \
     SimAppDialog \
     Telecom \
     TelephonyProvider \
-    TeleService \
     Traceur \
     UserDictionaryProvider \
     VpnDialogs \
-- 
2.17.1

